restapi/main.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"github.com/danielpaulus/go-ios/restapi/api"
+	_ "github.com/danielpaulus/go-ios/restapi/docs"
```

Comment:
This folder is currently on .gitignore so build fails. Easily fixed by removing the import since its not used but shouldn't we push the `docs` folder to the repo?

Reply:
It is auto generated by go-swagger. At some point a Makefile or sth similar should probably be provided. 
Take a look here: https://github.com/danielpaulus/go-ios/blob/add/basicAPI/.github/workflows/deploy.yml
at the preliminary build config.
TL;DR
run
`go install github.com/swaggo/swag/cmd/swag@latest
          && swag init`
and it will work.

Reply:
My bad, I didn't check the workflow, but I already had swaggo so just generated them locally for myself. Good stuff!